### DISCUSSION TITLE
Adding Author image automatically from wikipedia (or other sources)

### DISCUSSION THREAD

**Discussion by lukethehawk** [advplyr/audiobookshelf#1620] [Ideas]
Right now the auto image search does not work very well for me, it find very few images so I have to manually set the urls.
Would it be possible to add the automatic insertion of the image from wikipedia (if any)?

**Comment by advplyr** [comment]
Most likely not. We need a stable and reliable API to get author images from and currently the only one available is Audnexus (which scrapes Audible).

**Reply by lukethehawk** [reply]
Ok thanks, I hope you will find a new and more reliable API for that in the future

**Reply by tfranken90** [reply]
@advplyr would audiodb be a viable alternative for author information? https://www.theaudiodb.com/

**Reply by advplyr** [reply]
Yeah possibly. I searched a few authors real quick and was able to get some data
